waspc/src/Wasp/Analyzer/Evaluator/Evaluation/TypedExpr/Combinators.hs

```diff
@@ -152,7 +153,21 @@ tuple4 eval1 eval2 eval3 eval4 = evaluation $ \(typeDefs, bindings) -> withCtx $
 -- | An evaluation that expects an "ExtImport".
 extImport :: TypedExprEvaluation AppSpec.ExtImport.ExtImport
 extImport = evaluation' . withCtx $ \ctx -> \case
-  TypedAST.ExtImport name file -> pure $ AppSpec.ExtImport.ExtImport name file
+  TypedAST.ExtImport name fileStr ->
+    -- NOTE(martin): This parsing here could instead be done in Parser.
+    --   I don't have a very good reason for doing it here instead of Parser, except
+    --   for being somewhat simpler to implement.
+    --   So we might want to move it to Parser at some point in the future, if we
+    --   figure out that is better (it sounds/feels like it could be).
+    if take 5 fileStr /= "@ext/"
```

Comment:
That's a very good point! I added `where extPrefix = "@ext/" and replaced all usages of it with `extPrefix`. I also used `isPrefixOf` instead of `take` and `/=`.

Reply:
Correction -> I actually ended up using `stripPrefix` as it also removed it for me at the same moment.